packages/nextjs/src/index.server.ts

```diff
@@ -20,6 +19,17 @@ export { ErrorBoundary, withErrorBoundary } from '@sentry/react';
 type GlobalWithDistDir = typeof global & { __rewriteFramesDistDir__: string };
 const domain = domainModule as typeof domainModule & { active: (domainModule.Domain & Carrier) | null };
 
+// During build, the main process is invoked by
+//   `node next build`
+// and child processes are invoked as
+//   `node <path>/node_modules/jest-worker/build/workers/processChild.js`,
+// whereas at runtime the process is invoked as
+//   `node next start`
+// or
+//   `node /var/runtime/index.js`.
+const isBuild = new RegExp('build').test(process.argv.toString());
+const isVercel = !!process.env.VERCEL;
```

Comment:
Shall we look for `VERCEL=1` specifically? `!!process.env.VERCEL` means `isVercel` is true as long as the environment variable is defined (even for values that would be surprising to consider true, such as `VERCEL=0`, `VERCEL=no` and `VERCEL=false`).

https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables

Reply:
Let's ignore this for now as it's been like this for sometime already and we can address it later.